supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemJobCachingActorHelper.scala

```diff
@@ -1,45 +1,18 @@
 package cromwell.backend.sfs
 
-import akka.actor.{Actor, ActorRef}
+import akka.actor.Actor
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.backend.BackendInitializationData
-import cromwell.backend.callcaching.JobCachingActorHelper
-import cromwell.backend.io.{JobPathsWithDocker, WorkflowPathsBackendInitializationData}
-import cromwell.backend.validation.{RuntimeAttributesValidation, ValidatedRuntimeAttributes}
+import cromwell.backend.standard.{StandardCachingActorHelper, StandardInitializationData}
 import cromwell.core.logging.JobLogging
 import cromwell.core.path.PathBuilder
 import net.ceedubs.ficus.Ficus._
 
-trait SharedFileSystemJobCachingActorHelper extends JobCachingActorHelper {
+trait SharedFileSystemJobCachingActorHelper extends StandardCachingActorHelper {
```

Comment:
I get confused, is the distinction between SFS and Config backend still useful? Does anyone else extend the Config backend?